### ISSUE TITLE
185.228.250.0 has a location according to v1.5.1 "-t" but it's not in the resulting .csv

### ISSUE THREAD

**Issue by sgteq** [massimocandela/geofeed-finder#19]
Context: v1.5.1 finds 53 fewer feed URLs. The issue probably affect more than a single case.

```
sgteq@desk:~/geofeed$ ./geofeed-finder-linux-x64-1.5.1 -i ripe -o ripe.csv -k | tee ripe.log
...
Done! See ripe.csv
sgteq@desk:~/geofeed$ ./geofeed-finder-linux-x64-1.5.1 -t 185.228.250.0
inetnum: 185.228.250.0/24 https://128bit.ee/geofeed.csv [cache]
185.228.250.0/24,IT,IT-MI,Milan,

sgteq@desk:~/geofeed$ grep 185.228.250.0 ripe.csv
sgteq@desk:~/geofeed$ grep 128bit.ee ripe.log
```

Compare that to 1.5:

```
sgteq@desk:~/geofeed$ ./geofeed-finder-linux-x64-1.5 -i ripe -o ripe.csv -k | tee ripe.log
...
Done! See ripe.csv
sgteq@desk:~/geofeed$ ./geofeed-finder-linux-x64-1.5 -t 185.228.250.0
inetnum: 185.228.250.0/24 https://128bit.ee/geofeed.csv [cache]
185.228.250.0/24,IT,IT-MI,Milan,

sgteq@desk:~/geofeed$ grep 185.228.250.0 ripe.csv
185.228.250.0/24,IT,IT-MI,Milan,
sgteq@desk:~/geofeed$ grep 128bit.ee ripe.log
inetnum: 185.228.250.0/24 https://128bit.ee/geofeed.csv [download]
```

**Comment by massimocandela** [comment]
Hi @sgteq,

Thanks for reporting this. I fixed this in [v1.5.2](https://github.com/massimocandela/geofeed-finder/releases/tag/1.5.2), which is still in pre-release. I repeated your tests, but another pair of eyes are always better. Let me know. Thanks!

**Comment by sgteq** [comment]
Hi, @massimocandela,

v1.5.2 looks good to me. The fix recovered most of the lost feed URLs. v1.5 on 8/30 vs v1.5.2 on 9/5:
```
all-2022-08-30.log: 299 URLs 4,166,912 IPv4 addresses
all-2022-09-05.log: 287 URLs 4,012,986 IPv4 addresses
```
The rest of lost URLs except one are either due to a deliberate removal by maintainers or due to non-compliant remark formatting. I'm submitting one minor remaining issue separately. 

```
103.176.189.0.txt:remarks:        Geofeed: https://www.daryllswer.com/geofeed/
115.69.0.0.txt:remarks:        geofeed: https://bitbucket.org/spintelaustralia/geofeed/raw/master/geofeed.csv
194.82.115.0.txt:remarks:        Geofeed: https://resource.net.strath.ac.uk/geo/unistrath.csv
198.148.78.0.txt:Comment:        Geofeed  https://talosops-public.s3.amazonaws.com/talos.csv
199.101.72.0.txt:Comment:        Geofeed: https://viking.skywaywest.net/geofeed/SKYWAY-CENTRAL-1.csv
208.74.40.0.txt:Comment:        Geofeed is http://geoip.norvado.net/geofeed
216.239.128.0.txt:Comment:        Geofeed: https://www.omnis.com/geofeed.csv
2406_840_eb00__.txt:remarks:        geofeed: https://geofeed.zhiccc.net/2406-840-geofeed.csv
2406_840__.txt:remarks:        geofeed: http://net.zxinc.org/geofeed.csv
31.135.2.0.txt:remarks:        Geofeed: http://dyon-network.ro/geofeed/google.csv
86.105.9.0.txt:remarks:        Geofeed found at https://geoip.m247.ro/geofeeds.csv
```
